README.md

```diff
@@ -12,19 +12,38 @@
 </p>
 
 <p align="center">
-	<img src="https://raw.githubusercontent.com/catppuccin/foliate/main/assets/screenshot.webp"/>
+	<img src="assets/preview.webp"/>
 </p>
 
+## Previews
+
+<details>
+<summary>🌻 Latte</summary>
+<img src="assets/latte.webp"/>
+</details>
+<details>
+<summary>🪴 Frappé</summary>
+<img src="assets/frappe.webp"/>
+</details>
+<details>
+<summary>🌺 Macchiato</summary>
+<img src="assets/macchiato.webp"/>
+</details>
+<details>
+<summary>🌿 Mocha</summary>
+<img src="assets/mocha.webp"/>
+</details>
```

Comment:
All the assets in the assets folder start with a capital letter (e.g. `Latte.webp`) so those images need to be renamed to be lowercase.